### PR TITLE
Improve links and markup formatting

### PR DESCRIPTION
**Pull request by emgsilva**
Some cleaning 🙂 Improved some links (C4 related links with link to the specific diagram type referred; Michael's Bounded Contexts presentation link). Formatted a bit the markup (a lot of warnings). Added link to open issue from README (like we did on core-domain-charts). Also noticed that there is a missing figure on resources directory (event-storm-bounded-context.jpg), which leads to a broken link - can you fix that?

### CODE REVIEW

**Comment by cakper** [comment]
@emgsilva thank you for contribution - it's good to go. Before merging - can you please add yourself to the contributors section at the bottom? :) Thanks!

**Comment by emgsilva** [comment]
@cakper done + fixed the link to context maps link to Michael (was pointing to page 33 - I suppose that was not on purpose right?).

**Comment by cakper** [comment]
Sounds good. Thanks - I will merge now and rearrange the contributor list to keep it alphabetical :)

**Comment by cakper** [comment]
@emgsilva actually there is a conflict merged into your branch: https://github.com/ddd-crew/ddd-starter-modelling-process/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8R151-R155 can you please fix?

File: README.md [hunk 1]
```diff
@@ -104,42 +124,53 @@ As a starting point, we recommend [EventStorming](https://www.eventstorming.com/
...
+
 - reduced cognitive load, so that we can reason about parts of the domain independently,
 - give development teams autonomy, so that they can work on separate parts of the solution,
 - identifying loose-coupling and high-cohesion in the domain which carries over to our software architecture and team structure.
 
+<<<<<<< HEAD
+As a starting point, we recommend [carving up your event storm](resources/event-storm-bounded-context.jpg) into sub-domains and [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps).
+=======
 As a starting point, we recommend carving up your event storm into sub-domains and [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps?slide=33).
+>>>>>>> upstream/master
```
Comment (cakper) [inline README.md:155]: This needs fixing

Comment (cakper) [inline README.md:155]: This needs fixing.

Comment (emgsilva) [reply README.md]: Done @cakper, forgot to add the actual change I made on previous commit :p

**Pull request merged**